lib/fastlane/plugin.rb

```diff
@@ -0,0 +1,18 @@
+module Fastlane
+  module Plugin
+    module ClassMethods
+      def repository
+        @repository ||= []
+      end
+
+      def inherited(klass)
+        repository << klass
+        puts "Loaded Plugin #{klass}" #TODO: In the real implementation we would make this only happen to those loaded via fastfile
```

Comment:
Missing space after #.<br>Do not write to stdout. Use Rails' logger if you want to log.